pkg/ffmpeg/types.go

```diff
@@ -27,8 +27,14 @@ type FFProbeJSON struct {
 			Comment          string        `json:"comment"`
 		} `json:"tags"`
 	} `json:"format"`
-	Streams []FFProbeStream `json:"streams"`
-	Error   struct {
+	Streams          []FFProbeStream `json:"streams"`
+	PacketsAndFrames []struct {
+		Type         string `json:"type"`
+		SideDataList []struct {
+			Rotation int `json:"rotation"`
```

Comment:
The rotation data is not consumed. Do we need to check the value [here](https://github.com/stashapp/stash/pull/4781/files#diff-2146c23c42b5e16caf80faaefc0d7038e62cf4cf04e38e9502c8eff71e786b24R249) and set the weight and height accordingly?

Reply:
ugh, I knew I forgot to copy a line!